rubygems-update.gemspec

```diff
@@ -6,14 +6,15 @@ Gem::Specification.new do |s|
   s.authors = ["Jim Weirich", "Chad Fowler", "Eric Hodel", "Luis Lavena", "Aaron Patterson", "Samuel Giddins", "André Arko", "Evan Phoenix", "Hiroshi SHIBATA"]
   s.email = ["", "", "drbrain@segment7.net", "luislavena@gmail.com", "aaron@tenderlovemaking.com", "segiddins@segiddins.me", "andre@arko.net", "evan@phx.io", "hsbt@ruby-lang.org"]
 
-  s.summary = "RubyGems is a package management framework for Ruby."
+  s.summary = "RubyGems is a package management framework for Ruby. This gem is downloaded and installed by `gem update --system`, so that the `gem` CLI can update itself."
   s.description = "A package (also known as a library) contains a set of functionality
   that can be invoked by a Ruby program, such as reading and parsing an XML file. We call
   these packages 'gems' and RubyGems is a tool to install, create, manage and load these
   packages in your Ruby environment. RubyGems is also a client for RubyGems.org, a public
   repository of Gems that allows you to publish a Gem that can be shared and used by other
   developers. See our guide on publishing a Gem at guides.rubygems.org"
-  s.homepage = "https://rubygems.org"
+  s.homepage = "https://guides.rubygems.org"
+  s.metadata    = { "source_code_uri" => "https://github.com/rubygems/rubygems" }
```

Comment:
:thinking: there is a lot of spaces

Reply:
I swear I am normally a very good programmer but not today. Force-pushed to fix this :(